resource/resource.h

```diff
@@ -106,19 +106,16 @@
 #define IDC_OPEN_NOTIFICATION           1086
 #define IDC_EDIT1                       1087
 #define IDC_SRD_NOTES                   1087
-#define IDC_OPEN_NOTIFICATION2          1087

```

Comment:
Intentional?

Reply:
Yep - its a button I added and then got rid of, but VSisn't clever enough to remove from the header!